### PR TITLE
config: Add `purchaseaccount` to sample config.

### PR DESCRIPTION
**Pull request by amass01**
This diff adds missing `purchaseaccount` to `sample-dcrwallet.conf`.

### CODE REVIEW

File: sample-dcrwallet.conf [hunk 1]
```diff
@@ -23,6 +23,9 @@
 ; flag.
 ; enablevoting=0
 
+; Account to autobuy tickets from.
+; purchaseaccount=
```
Comment (amass01) [inline sample-dcrwallet.conf:27]: Actually this is an autobuyer config therefore should be moved to `ticketbuyer.purchaseaccount`.

@jrick would it make sense to make this change in a separate PR ?

Comment (jrick) [reply sample-dcrwallet.conf:27]: no, that will break existing configurations

Comment (amass01) [reply sample-dcrwallet.conf:27]: yep, thought about that, then we leave it for the right moment
